{"name": "test_get_record", "status": "broken", "statusDetails": {"message": "TypeError: 'int' object is not callable", "trace": "self = <tests.test.crud.test_crud.TestCrud object at 0x0000019C6ABAC5E0>\nget_booking_id = 843\n\n    def test_get_record(self,get_booking_id):\n        booking_id=get_booking_id\n        url=APIConstants.url_get_search(booking_id=booking_id)\n        response=get_request(url=url,auth=None,in_json=False)\n        firstname=response.json()['firstname']\n>       status=response.status_code()\nE       TypeError: 'int' object is not callable\n\ntest_crud.py:51: TypeError"}, "start": 1730116360265, "stop": 1730116362059, "uuid": "1f46ea54-b344-4738-88b4-9fa0acdc3840", "historyId": "6407ee2c92658c759a908696ec8b1c02", "testCaseId": "a97a0954cbddcda4dd989c232a4b2145", "fullName": "tests.test.crud.test_crud.TestCrud#test_get_record", "labels": [{"name": "parentSuite", "value": "tests.test.crud"}, {"name": "suite", "value": "test_crud"}, {"name": "subSuite", "value": "TestCrud"}, {"name": "host", "value": "LAPTOP-J82A4UMN"}, {"name": "thread", "value": "13944-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test.crud.test_crud"}]}